### PR TITLE
Curator Update

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
:cl: Kor, Profakos, Iamgoofball
add: The Librarian has been replaced by the Curator.
add: Persistent trophy cases have been added to the library on Meta, Delta, and Box. These are only usable by the Curator.
add: The Curator now starts with his whip.
add: The Curator now has access to his treasure hunter outfit, stashed in his private office.
/:cl:

Why: All of this has been debated and approved individually, just finally adding it to the maps. Renamed the Librarian to Curator to reflect his new job duties and to """rebrand""" the job away from being a miserable joke.

The only change is that the librarian alone can use the display cases because:

-They are single use
-They are in a public area

Don't want his job gimmick to be stolen by assistants or the captain (if we used an ID lock)

### CODE REVIEW

File: code/game/objects/structures/displaycase.dm [hunk 1]
```diff
@@ -278,6 +278,9 @@
 	if(!user.Adjacent(src)) //no TK museology
 		return
 
+	if(!(user.mind && user.mind.assigned_role == "Librarian"))
+		to_chat(user, "<span class='danger'>You're not sure how to work this. Maybe you should as the librarian for help.</span>")
```
Comment (ChangelingRain) [inline code/game/objects/structures/displaycase.dm]: ask
and you need a return

Comment (KorPhaeron) [inline code/game/objects/structures/displaycase.dm]: !

**Comment by Bawhoppen** [comment]
:-1: suit being in bookbag WHAT THE ABSOLUTE FUCK

**Comment by coiax** [comment]
I'm not a fan of items being inside storage containers that normally wouldn't accept them.

File: code/game/objects/items/weapons/storage/bags.dm [hunk 2]
```diff
@@ -291,6 +291,16 @@
 	can_hold = list(/obj/item/weapon/book, /obj/item/weapon/storage/book, /obj/item/weapon/spellbook)
 	resistance_flags = FLAMMABLE
 
+/obj/item/weapon/storage/bag/books/curator/New()
+	..()
```
Comment (ChangelingRain) [inline code/game/objects/items/weapons/storage/bags.dm]: dude no use the proc specifically for filling bags, PopulateContents()

**Comment by KorPhaeron** [comment]
Open to other suggestions as to were to put it

File: code/game/objects/items/weapons/storage/bags.dm (hunk 2 above)
Comment (KorPhaeron) [inline code/game/objects/items/weapons/storage/bags.dm]: I am very out of touch

**Comment by Bawhoppen** [comment]
just add it to the map, you already touched them

**Comment by coiax** [comment]
@KorPhaeron it would make sense for librarians to be able to pick their curating outfit with outfitting points, if and when I implement that

**Comment by KorPhaeron** [comment]
There isnt really space in the backroom and its horribly insecure there anyway, assistants would take it every round

**Comment by coiax** [comment]
Why not just put it in their backpack

**Comment by GunHog** [comment]
Why not Cere as well?

**Comment by Profakos** [comment]
*single tear of joy*

**Comment by Tacolizard** [comment]
you fucked up the cl

**Comment by Iamgoofball** [comment]
@KorPhaeron just make them wear the curator's suit and rename them to Curator

**Comment by Niknakflak** [comment]
no

**Comment by MMMiracles** [comment]
I think you may of forgotten Cerestation in there.

File: code/game/objects/items/weapons/storage/bags.dm (hunk 2 above)
Comment (Cyberboss) [inline code/game/objects/items/weapons/storage/bags.dm]: >ADDING A NEW

Kor are u tryna fucking kill me?

Comment (KorPhaeron) [inline code/game/objects/items/weapons/storage/bags.dm]: I copied my 4 month old PR without thinking about it gomenesai

**Comment by KorPhaeron** [comment]
@MMMiracles your library has a weird layout and I couldn't figure it out quickly and I figured you'd be able to do it anyway since the map still has someone actively dedicated to it

File: code/game/objects/items/weapons/storage/bags.dm [hunk 3]
```diff
@@ -291,6 +291,16 @@
 	can_hold = list(/obj/item/weapon/book, /obj/item/weapon/storage/book, /obj/item/weapon/spellbook)
 	resistance_flags = FLAMMABLE
 
+/obj/item/weapon/storage/bag/books/curator/New()
+	..()
+	new /obj/item/clothing/shoes/workboots/mining(src)
+	new /obj/item/clothing/head/curator(src)
+	new /obj/item/clothing/suit/curator(src)
+	new /obj/item/clothing/under/rank/librarian/curator(src)
+	new /obj/item/weapon/melee/curator_whip(src)
```
Comment (kevinz000) [inline code/game/objects/items/weapons/storage/bags.dm]: op pls nerf

Comment (PKPenguin321) [inline code/game/objects/items/weapons/storage/bags.dm]: leave out until reach weapons are unfucked then make it a super cool reach weapon

Comment (GunHog) [inline code/game/objects/items/weapons/storage/bags.dm]: Relax Kev, let the boomworm have his Indiana Jones fantasy!

**Comment by Repukan** [comment]
Put it in that little cuck room they get

**Comment by Bawhoppen** [comment]
Yea seriously just add it to the map, It's not that hard

**Comment by Cyberboss** [comment]
This was not ready

**Comment by Incoming5643** [comment]
I disagree with using assigned role as a lockout for anything, ever. It's an intrinsic job effect that can not be replicated in game by any means other than being the librarian to start with.

**Comment by YourNeighborsObnoxiousCat** [comment]
Items can be accidentally placed inside the cases. Perhaps a quick Yes/No dialogue could help with this?

**Comment by KorPhaeron** [comment]
@Incoming5643 

Generally I agree to the point of not wanting ID locks on machines at all but this is entirely cosmetic

**Comment by Incoming5643** [comment]
I have a solution that I think will work for both of us, just a bit

**Comment by Bawhoppen** [comment]
wait this renamed librarian? wtf thats dumb

**Pull request merged**